specs/DesignDiscussions/InputLabels.md

```diff
@@ -72,17 +72,55 @@ This conflict speaks to a general tension inherent in the design of Adaptive Car
 
 In order to gather more data to help us balance the needs of the card authors and hosts, we will send a survey to customers who have expressed an interest in improving out input story. The following options are under consideration, and will be evaluated against survey data to determine which if any will be included in our v1 implementation.
 
+> ## Rendering for string labels
+>
+> The default rendering for labels should be performed with the default values for all of the following properties:
```

Comment:
### Card Author Formatting Support
1. Simple Text String in Labels
2. Formatting support in Labels
- TextBlock version

.......................
<Final image card>